apps/admin/backend/src/globals.ts

```diff
@@ -57,3 +58,10 @@ const DEFAULT_ALLOWED_EXPORT_PATTERNS =
 export const ADMIN_ALLOWED_EXPORT_PATTERNS =
   process.env.ADMIN_ALLOWED_EXPORT_PATTERNS?.split(',') ??
   DEFAULT_ALLOWED_EXPORT_PATTERNS;
+
+/**
+ * Print settings for reports, namely one-sided printing.
+ */
+export const REPORT_PRINT_OPTIONS: PrintOptions = {
+  sides: PrintSides.OneSided,
+};
```

Comment:
Is there any situation in VxSuite where we want to print double-sided? If not, we can just hardcode this into libs/printing instead of making it a parameter, which would solve your problem of making sure it's always single-sided.

Or a weaker version would just be to change the default to single-sided.

Reply:
This solution works and is much simpler! Tested on VxAdmin and VxPollBook.